### PR TITLE
Spring boot generated project

### PR DESCRIPTION
**Pull request by DuckDrick**
Spring boot project generated using https://start.spring.io
Dependencies:
image

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation 'org.springframework.boot:spring-boot-starter-mail'
+	implementation 'org.springframework.boot:spring-boot-starter-security'
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.liquibase:liquibase-core'
+	compileOnly 'org.projectlombok:lombok'
+	runtimeOnly 'org.postgresql:postgresql'
+	annotationProcessor 'org.projectlombok:lombok'
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation 'org.springframework.security:spring-security-test'
```
Comment (PauliusBert) [inline build.gradle]: Not sure if it's related to Tests, but it seems so. I would like if you would remove everything that's needed for tests as we wont write any of them

**Review by PauliusBert — CHANGES_REQUESTED** [review]

**Review by saltanovas — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (saltanovas) [reply build.gradle]: doesn't bother me :)

Comment (PauliusBert) [reply build.gradle]: Well, those two lines doesn't bother me either, but this class/folder does.
image

Comment (saltanovas) [reply build.gradle]: Whatever you choose, resolve the conversation. I made a restriction that every conversation must be resolved before merge.

**Review by StauceLukas — APPROVED** [review]
I would agree with @HabaHaba05 that tests folder should be removed.

**Review by PauliusBert — APPROVED** [review]

**Pull request merged**